dkg/messages.go

```diff
@@ -150,6 +153,43 @@ func (msg *Init) Decode(data []byte) error {
 	return json.Unmarshal(data, msg)
 }
 
+// Reshare triggers the resharing protocol
+type Reshare struct {
+	// ValidatorPK is the the public key to be reshared
+	ValidatorPK types.ValidatorPK
+	// OperatorIDs are the operators in the new set
+	OperatorIDs []types.OperatorID
+	// Threshold is the threshold of the new set
+	Threshold uint16
+}
+
+func (msg *Reshare) Validate() error {
+
+	if len(msg.ValidatorPK) != blsPubkeySize {
+		return errors.New("invalid validator pubkey size")
+	}
+
+	if len(msg.OperatorIDs) < 4 || (len(msg.OperatorIDs)-1)%3 != 0 {
+		return errors.New("invalid number of operators which has to be 3f+1")
+	}
+
+	if int(msg.Threshold) != (len(msg.OperatorIDs)-1)*2/3+1 {
```

Comment:
Let's change operator count validation to 4,7,10,13 and threshold accordingly

Reply:
Sure. Will do.